hstrat/_auxiliary_lib/_HereditaryStratigraphicInstrumentation.py

```diff
@@ -0,0 +1,18 @@
+import typing
+
+if typing.TYPE_CHECKING:
+    from ..genome_instrumentation import (
+        HereditaryStratigraphicColumn,
+        HereditaryStratigraphicSurface,
+    )
+
+HereditaryStratigraphicInstrumentation = typing.Union[
```

Comment:
What's the difference between `HereditaryStratigraphicInstrumentation` and `HereditaryStratigraphicInstrumentation_T`? I'm not familiar with this pattern!